core/controllers/acl_decorators.py

```diff
@@ -568,7 +568,7 @@ def test_super_admin(self, **kwargs):
         if not self.user_id:
             raise self.NotLoggedInException
 
-        if not user_services.is_current_user_super_admin():
+        if not self.is_super_admin:
```

Comment:
I think that, by now, we're quite used to assuming that methods starting "is_" are methods in this codebase. Can you maybe make this a method too? You can have a self._current_user_is_super_admin private property.

Or you can set and use self.current_user_is_super_admin directly.

Reply:
Done.